src/atoms/Text/__tests__/Text.web.test.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { renderWithTheme } from '../../../_helpers/testing';
+
+import Text from '../index';
+
+describe('Renders Text', () => {
+  it('snapshot testing', () => {
```

Comment:
where are different variants test? Refer native tests and update it

Reply:
We'll have a discussion today on testing

Reply:
done